samcli/commands/pipeline/init/interactive_init_flow.py

```diff
@@ -256,7 +256,7 @@ def _generate_from_pipeline_template(
                 continue
             click.echo(
                 Colored().yellow(
-                    "2 stage(s) were detected, matching the template requirements. "
+                    f"{number_of_stages} stage(s) were detected, matching the template requirements. "
```

Comment:
Can we update existing unit tests to confirm this message from now on?

Reply:
let me check the existing unit test cases.